src/Shared/Contracts/IManagerPackageActions.cs

```diff
@@ -38,25 +38,25 @@ public interface IManagerPackageActions<T> where T : IManagerPackageVersionMetad
     Task<bool> DoesPackageExistAsync(PackageURL packageUrl, bool useCache = true, CancellationToken cancellationToken = default);
 
     /// <summary>
-    /// Gets all versions of a package.
+    /// Gets all versions of a package, both releases and pre-releases are included by default.
     /// </summary>
     /// <remarks>The first version in the list is the latest, and the oldest version is the last.</remarks>
     /// <param name="packageUrl">The <see cref="PackageURL"/> to get all available versions for.</param>
-    /// <param name="includePrerelease">If pre-release/beta versions should be included.</param>
+    /// <param name="includePrerelease">If pre-release/beta versions should be included, defaults to <c>true</c>.</param>
     /// <param name="useCache">If the cache should be checked for the available versions of this package.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to be used in the method call, defaults to <see cref="CancellationToken.None"/>.</param>
     /// <returns>An <see cref="IEnumerable{String}"/> of all of the versions for this package, in descending order.</returns>
     Task<IEnumerable<string>> GetAllVersionsAsync(PackageURL packageUrl, bool includePrerelease = true, bool useCache = true, CancellationToken cancellationToken = default);
 
     /// <summary>
-    /// Gets the latest version of a package.
+    /// Gets the latest version of a package, both releases and pre-releases are included.
+    /// TODO: Reimplement include pre-release when all project managers support it.
     /// </summary>
     /// <param name="packageUrl">The <see cref="PackageURL"/> to get the latest version for.</param>
-    /// <param name="includePrerelease">If pre-release/beta versions should be included.</param>
     /// <param name="useCache">If the cache should be checked for the latest of this package.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to be used in the method call, defaults to <see cref="CancellationToken.None"/>.</param>
     /// <returns>The latest version of the <paramref name="packageUrl"/> or null if there are no versions.</returns>
-    Task<string?> GetLatestVersionAsync(PackageURL packageUrl, bool includePrerelease = false, bool useCache = true, CancellationToken cancellationToken = default);
+    Task<string?> GetLatestVersionAsync(PackageURL packageUrl, bool useCache = true, CancellationToken cancellationToken = default);
```

Comment:
Changing this to take an options object also has the advantage of future changes to add more arguments not breaking compatibility.

Reply:
> Changing this to take an options object also has the advantage of future changes to add more arguments not breaking compatibility.

Yeah, that's an interesting point. It seems that [the convention in C# is that the CancellationToken always goes last](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1068)? That convention unfortunately gets in the way of adding new optional arguments to the end / rightmost position of a method signature, which would otherwise be fine.